flutter_local_notifications_linux/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: flutter_local_notifications_linux
+description: Linux implementation of the flutter_local_notifications plugin
+version: 0.1.0
+homepage: https://github.com/MaikuB/flutter_local_notifications/tree/master/flutter_local_notifications
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_local_notifications_platform_interface: ^4.0.0
+  dbus: ^0.5.0
+  path: ^1.8.0
+  posix: ^2.1.0
+  xdg_directories: ^0.2.0
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  mocktail: ^0.1.4
```

Comment:
The problem with Mockito is that it requires generating mock-files using `build_runner`. This was used after Dart announced null-safety, because now we can't use stubs as before if we are using inheritance from the `Mock` class. Stubs are `Null` type, while method arguments can be non-nullable:
![2021-07-25 18-14-25](https://user-images.githubusercontent.com/7840559/126904749-14006d86-5756-4967-b6d4-43181ab205b9.png)

Mocktail offers a "good old" API that is minimally different from Mockito and requires few changes to migrate. If that's important, I can use Mockito, but then `build_runner` will become part of the build process.

Reply:
Yep i've heard of the issues. This was more to point out that there'd be some inconsistency that could be addressed later, which may mean the main plugin moves to mocktail too. Personally, I'm not concerned about build_runner being part of the process and first-party plugins still use mockito